configs/odoo-server.conf

```diff
@@ -49,11 +52,11 @@ reportgz = False
 secure_cert_file = {{SSL_CERT}}
 secure_pkey_file = {{SSL_KEY}}
 server_wide_modules = None
-smtp_password = {{EMAIL_PASS}} ;False
-smtp_port = 465  ;25
-smtp_server = {{EMAIL_SERVER}}   ;localhost
-smtp_ssl = True                  ;False
-smtp_user = {{EMAIL_USER}}       ;False
+smtp_password = {{EMAIL_PASS}}   #False
+smtp_port = 465                  #25
+smtp_server = {{EMAIL_SERVER}}   #localhost
+smtp_ssl = True                  #False
+smtp_user = {{EMAIL_USER}}       #False
```

Comment:
Actually, this formatting will be broken after replacing placing placeholders to the values.

Reply:
I proposed that because there is problems to run Odoo with the lines commented by the ";" like for filter and mail option, for it this is an error and so the service is not launched requiring to modify the config file each time.

Reply:
I didn't face any problems with semi-colon comments in config...

Reply:
Could you only change semicolon to ``#`` only without further updates? I will merge it then

Reply:
Tested on Ubuntu 17.10. 

With semi-columns, after a first service start it is directly exited with a failure status and in the logs you got by example `
Dec 28 14:57:52 Ubuntu odoo-server[8862]: optparse.OptionValueError: option smtp_port: invalid integer value: '465  ;25'`"
'
If I let the default config with semi columns for the dbfilter , after a database creation the screen remains on the first DB creation.

Reply:
If I let the dbfilter option like in the source : `
dbfilter = ^%h$  #select database automatically by domain. ...
` it doesn't work too, the value "^%h$" is not recognized and the page stay to the DB creation. So it's why I proposed a first commented line with the explanation and the second line with only the parameter like excepted by the service as you can find it in many configurations (not like a commented script).